applications/FluidDynamicsApplication/tests/test_fluid_computation_processes.py

```diff
@@ -0,0 +1,82 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.KratosUnittest as UnitTest
+from KratosMultiphysics.testing.utilities import ReadModelPart
+from KratosMultiphysics.process_factory import KratosProcessFactory
+
+class FluidComputationProcessTest(UnitTest.TestCase):
```

Comment:
```suggestion
class FluidComputationProcessesTest(UnitTest.TestCase):
```
To be consistent with the file name and make clear that we will test similar auxiliary Python modules in here.